common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -243,9 +241,7 @@ public static ServerSocket getSocket(final TServerSocket thriftSocket) {
   public static InetSocketAddress getMasterAddress(TachyonConf conf) {
     String masterHostname =
         conf.get(Constants.MASTER_HOSTNAME, getLocalHostName(conf));
-    // Cannot rely on tachyon-default.properties because GetMasterWorkerAddressTest will test with
-    // fake conf
-    int masterPort = conf.getInt(Constants.MASTER_PORT, Constants.DEFAULT_MASTER_PORT);
```

Comment:
Does this mean DEFAULT_MASTER_PORT can be removed? (applies to other constants which were just default values)